api/v1alpha1/validation.go

```diff
@@ -28,10 +31,27 @@ func (o *EtcdCluster) ValidateDelete() error {
 }
 
 // ValidateCreate validates that only supported fields are changed
-// TODO: Not yet implemented
 func (o *EtcdCluster) ValidateUpdate(old runtime.Object) error {
-	var allErrs field.ErrorList
-	return allErrs.ToAggregate()
+	oldO, ok := old.(*EtcdCluster)
+	if !ok {
+		return fmt.Errorf("Unexpected type for old: %#v", old)
+	}
+	oldO = oldO.DeepCopy()
+
+	if *oldO.Spec.Replicas > *o.Spec.Replicas {
+		return fmt.Errorf(
+			"Unsupported changes: spec.replicas: current: %d, new: %d: scale-in is not supported",
+			*oldO.Spec.Replicas, *o.Spec.Replicas,
+		)
+	}
```

Comment:
is this not how we expect to scale clusters?

Reply:
discussed offline - this is just example usage for now, we'll likely remove this when scaling is implemented